### PR TITLE
Add remix example

### LINKED ISSUE #47: Use StepZen with Remix
Use StepZen in a Remix application https://remix.run/

### PR DESCRIPTION
**Pull request by tiluckdave**
### Fixes #47 

StepZen example with https://remix.run

@royderks

### CODE REVIEW

File: with-remix/app/root.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  var requestOptions = {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: graphql,
+  };
+
+  const res = await fetch('https://graphql8f.stepzen.net/api/8f0ff813c3676d81a04a616ba555af0c/__graphql', requestOptions);
```
Comment (royderks) [inline with-remix/app/root.jsx:31]: Could you include the setup instructions for the GraphQL API in the README?

File: with-remix/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Welcome to Remix!
+
+- [Remix Docs](https://remix.run/docs)
```
Comment (royderks) [inline with-remix/README.md:3]: We have a template for the README, it can be found in the file TEMPLATE.md

Please have a look and adjust accordingly :)

**Review by royderks — CHANGES_REQUESTED** [review]

**Comment by tiluckdave** [comment]
I have updated the README.md file

**Review by royderks — APPROVED** [review]

**Comment by tiluckdave** [comment]
Hey, @royderks am I eligible for these goodies 
image

**Comment by royderks** [comment]
Yes @tiluckdave, Make sure to join our [Discord](https://discord.com/invite/9k2VdPn2FR) so we can help get the stickers over to you

**Pull request merged**